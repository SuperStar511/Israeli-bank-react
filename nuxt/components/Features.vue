<template>
  <section
    id="features"
    class="bg-white border-b py-8"
  >
    <div class="container max-w-5xl mx-auto m-8">
      <section-title>תכונות</section-title>
      <feature-right
        headline="ייבוא מאתר הבנק"
        content="בעו&quot;שי ניתן לייבא נתונים בצורה אוטומטית מפירוט העובר ושב של חשבונות הבנק והאשראי."
      >
        <img
          slot="svg"
          src="~assets/img/cloud_files.svg"
          alt
        >
      </feature-right>
      <feature-left
        headline="ייצוא לתוכנות ניהול"
        content="את הנתונים שעו&quot;שי מושך מחשבונות הבנק והאשראי ניתן לייצא לתוכנות ניהול נתונים שונות."
      >
        <img
          slot="svg"
          src="~assets/img/file_sync.svg"
          alt
        >
      </feature-left>
    </div>
  </section>
</template>

<script>
import FeatureRight from '@/components/FeatureRight';
import FeatureLeft from '@/components/FeatureLeft';
import SectionTitle from '@/components/SectionTitle';

export default {
  name: 'Features',
  components: {
    'feature-right': FeatureRight,
    'feature-left': FeatureLeft,
    'section-title': SectionTitle,
  },
};
</script>
