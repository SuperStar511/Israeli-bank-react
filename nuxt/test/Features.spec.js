import { shallowMount } from '@vue/test-utils';
import Features from '@/components/Features';

const factory = () => shallowMount(Features);

describe('Features', () => {
  it('is a Vue instance', () => {
    const wrapper = factory();
    expect(wrapper.isVueInstance()).toBeTruthy();
  });

  it('renders properly', () => {
    const wrapper = factory();
    expect(wrapper.html()).toMatchSnapshot();
  });
});
