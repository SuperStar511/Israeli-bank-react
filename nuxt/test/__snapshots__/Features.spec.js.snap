// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Features renders properly 1`] = `
"<section id=\\"features\\" class=\\"bg-white border-b py-8\\">
  <div class=\\"container max-w-5xl mx-auto m-8\\">
    <section-title-stub>תכונות</section-title-stub>
    <feature-right-stub headline=\\"ייבוא מאתר הבנק\\" content=\\"בעו&quot;שי ניתן לייבא נתונים בצורה אוטומטית מפירוט העובר ושב של חשבונות הבנק והאשראי.\\"><img src=\\"~assets/img/travel-booking.svg\\" alt=\\"\\"></feature-right-stub>
    <feature-left-stub headline=\\"ייצוא לתוכנות ניהול\\" content=\\"את הנתונים שעו&quot;שי מושך מחשבונות הבנק והאשראי ניתן לייצא לתוכנות ניהול נתונים שונות.\\"><img src=\\"~assets/img/connected-world.svg\\" alt=\\"\\"></feature-left-stub>
  </div>
</section>"
`;
